docs/userGuide/contentAuthoring.md

```diff
@@ -0,0 +1,92 @@
+<link rel="stylesheet" href="{{baseUrl}}/css/main.css">
+
+<include src="../common/header.md" />
+
+<div class="website-content">
+
+# Content authoring
+### General writing guide
+*MarkBind* allows you to mix HTML with Markdown code. Be sure to use new line to separate Markdown from HTML elements.
+e.g.
+
+Instead of
+
+```
+<Panel>
+**Markdown** Content
+</Panel>
+```
+
+use:
+
+```
+<Panel>
+
+**Markdown** Content
+
+</Panel>
+```
+

```

Comment:
Is this old?
I remembered we change it to mention `<md>` and `<markdown>` tags

Reply:
ah yes. I referred to an old version of the user guide. Will update accordingly!